.babelrc

```diff
@@ -8,7 +8,11 @@
   "env": {
     "test": {
       "presets": [
-        "@babel/preset-env",
+        ["@babel/preset-env", {
+          "targets": {
+            "browsers": "node >=10"
```

Comment:
this is kind of confusing to me. `browsers: node >=10` but node isn't a browser...? 

I'm also not clear on whether this is valid syntax or not? https://babeljs.io/docs/en/babel-preset-env#targets

Reply:
It's valid syntax - https://babeljs.io/docs/en/babel-preset-env#targetsbrowsers

And `node >= 10` is a valid browserslist query. See https://github.com/browserslist/browserslist#full-list. This is inside of the `test` babel env, which is the jest tests that are running in node.